previewer/metadata_renderer.js

```diff
@@ -45,6 +51,8 @@ setMetadataAssetID = function(ID) {
     )
 
     visibleProps.forEach(function(prop) {
+      if (!asset.hasOwnProperty(prop))
+        return;
```

Comment:
This is a tricky one - I like the idea that this panel shows all the things that could be filled in and not just what you have filled in already to help libingester devs to know what they are missing. Maybe we can have this list dynamically generate the rows based on the asset type and some consts in libingester iself?